awx/main/access.py

```diff
@@ -2264,7 +2264,7 @@ class JobEventAccess(BaseAccess):
     '''
 
     model = JobEvent
-    prefetch_related = ('hosts', 'children', 'job__job_template', 'host',)
```

Comment:
@matburt this is why I was looking at this. This prefetch_related scared me. In my case, I had 975 job event children... but who on Earth knows how many real users would have. This would load that many into memory any time the parent event was loaded.

Thankfully, this was never an issue because the relationship just happened to always be empty.